docs/source/whatsnew/pr/incompat-readline.rst

```diff
@@ -0,0 +1,4 @@
+IPython 5.0 now use prompt_toolkit, any setting that affect ``readline`` will
+have no effect, and has (likely) be relplaced by a configuration option on
```

Comment:
'has likely been replaced'

Well done on getting *effect* and *affect* the right way round, though - that trips up many native speakers.